media-gfx/alembic/alembic-1.7.16.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake multiprocessing
+
+DESCRIPTION="Open framework for storing and sharing scene data"
+HOMEPAGE="https://www.alembic.io/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="BSD"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="examples hdf5 prman test zlib"
+RESTRICT="!test? ( test )"
+
+REQUIRED_USE="hdf5? ( zlib )"
+
+RDEPEND="
+	>=media-libs/openexr-2.5.0:=
+	hdf5? ( >=sci-libs/hdf5-1.10.2:=[zlib(+)] )
+	zlib? ( >=sys-libs/zlib-1.2.11-r1 )
+"
+DEPEND="${RDEPEND}"
+BDEPEND=""
+
+DOCS=( "ACKNOWLEDGEMENTS.txt" "FEEDBACK.txt" "NEWS.txt" "README.txt" )
+
+PATCHES=( "${FILESDIR}/${PN}-1.7.11-0005-Fix-install-locations.patch" )
+
+src_configure() {
+	local mycmakeargs=(
+		-DALEMBIC_SHARED_LIBS=ON
+		# C++-11 and thus {shared,unique,weak}_ptr are common nowadays, so these
+		# are no longer needed and using boost fails. Options will get removed by
+		# upstream soon
+		-DALEMBIC_LIB_USES_BOOST=OFF
+		-DALEMBIC_LIB_USES_TR1=OFF
+		-DUSE_ARNOLD=OFF
+		-DUSE_BINARIES=ON
+		-DUSE_EXAMPLES=$(usex examples)
+		-DUSE_HDF5=$(usex hdf5)
+		-DUSE_MAYA=OFF
+		-DUSE_PRMAN=$(usex prman)
+		-DUSE_PYALEMBIC=OFF
+		-DUSE_TESTS=$(usex test)
+	)
+
+	cmake_src_configure
+}
+
+pkg_postinst() {
+	if use prman; then
+		einfo "If you're looking for an ebuild for renderman, you may want to"
+		einfo "try the waebbl overlay: 'eselect repository enable waebbl'"
+		einfo "followed by 'emerge renderman'"
+	fi
```

Comment:
Want to package that for ::gentoo as well?

Reply:
I thought about this several times. But the package has several characteristics, so I decided against it so far:
- it's a binary only package
- fetch restricted, user has to register and download manually
- license restricted to one year, use has to re-license every year
- blender integration, which was the reason, I build the package at first, is far behind blender development, currently there is only a renderman for blender-2.79, a version for 2.8+ is questionable, just checked their forum
- is there demand for a build of the package?
Probably a good idea to remove the flag too. What do you think?

Reply:
We've probably seen worse, but it does look like a mess I wouldn't want to welcome after all :P

Reply:
Yes it is. I'm removing the flag for it.